packages/@aws-cdk/cloudformation-include/README.md

```diff
@@ -88,6 +88,24 @@ const bucket = s3.Bucket.fromBucketName(this, 'L2Bucket', cfnBucket.ref);
 // bucket is of type s3.IBucket
 ```
 
+## Conditions
+
+If your template uses [CloudFormation Conditions](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/conditions-section-structure.html),
+you can retrieve them from your template:
+
+```typescript
+import * as core from '@aws-cdk/core';
+
+const condition: core.CfnCondition = cfnTemplate.getCondition('MyCondition');
```

Comment:
Wonder if it will be useful to also return a `listConditions()` API that returns an iterable of `CfnCondition`.

Not necessary for this PR, of course.

Reply:
Sure 🙂